benchmarks/up-arrow.bril

```diff
@@ -0,0 +1,36 @@
+# ARGS: 2 3 3
+
+@main(n: int, arrows: int, repeats: int) {
+  ans: int = call @up_arrow n arrows repeats;
+  print ans;
+}
+
+@up_arrow(num: int, arrows: int, repeats: int): int {
+  one: int = const 1;
+
+  ans: int = id num;  
+  i: int = const 1;
+
+.loopstart:
+  keepgoing: int = lt i repeats;
```

Comment:
`int` -> `bool`?

Reply:
Oh, good catch! Yeah brili still lets this work for some reason...

Reply:
Yeah, different implementations. I'm just trying to make sure `brilirs` runs on all of the new benchmarks that are being added and its more strict about these things.